op-batcher/batcher/channel_manager.go

```diff
@@ -273,3 +333,14 @@ func (s *channelManager) AddL2Block(block *types.Block) error {
 
 	return nil
 }
+
+func l2BlockRefFromBlockAndL1Info(block *types.Block, l1info derive.L1BlockInfo) eth.L2BlockRef {
```

Comment:
Shouldn't this be pulling in from `l2block_util.go`?

Reply:
We could but here we have the situation that we don't need and want to extract the `l1info` again, so made this local helper. The extraction is extra work which I saved here. But there's also an argument to favor simplicity.